src/StudyUtils.in.jsm

```diff
@@ -1,5 +1,27 @@
 "use strict";
 
+/**
+* OUTSTANDING QUESTIONS FOR GREGG
+*  - shield-studies-addon schema description references attributes sent as 
+*  Map(string, string). Can you explain what this means?
+*  - Can you describe the purpose of the `studySetup` and `webExtensionMsg` schema?
```

Comment:
They are the validators for what a valid  `.setup` options, and a what is a valid message that the utils can handle from the bound method from a `webExtension` can send.

Reply:
So just to confirm: `studySetup` schema validates the argument passed into `StudyUtils.setup()` and `webExtensionMsg` should be used to validate the first argument passed into `respondToWebExtensionMessage` in `StudyUtils.constructor`, but it doesn't look like it is used currently? Can you confirm?